packages/tenant-process/src/utilities/errorMappers.ts

```diff
@@ -60,10 +61,25 @@ export const selfcareUpsertTenantErrorMapper = (
     .with("selfcareIdConflict", () => HTTP_STATUS_CONFLICT)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
 
+export const addCertifiedAttributeErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("tenantNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with("tenantIsNotACertifier", () => HTTP_STATUS_FORBIDDEN)
+    .with("registryAttributeIdNotFound", () => HTTP_STATUS_NOT_FOUND)
```

Comment:
```suggestion
    .with("attributeNotFound", () => HTTP_STATUS_BAD_REQUEST)
```
This is currently returned as BAD REQUEST, maybe it's correct because the tenant exists while the attribute passed is incorrect.

What do you think? @galales @beetlecrunch